README.md

```diff
@@ -234,6 +235,40 @@ effect();
 
 [Try it](https://share.effector.dev/6VRR39iC)
 
+## Abort
+
+[Method documentation & API](/abort)
+
+```ts
+import { createEvent } from 'effector';
+import { abort } from 'patronum/abort';
+
+const cancel = createEvent<string>();
+const abortableFx = abort({
+  signal: cancel,
+  getKey: () => 'shared',
+  async handler(_: void, { onAbort }) {
```

Comment:
In my opinion, the most popular case for cancellable effects is a fetch-request. Maybe, we should provide an example for cancellable fetch here.